e2e/commands/watch.e2e.2.ts

```diff
@@ -41,7 +41,7 @@ describe('bit watch command', function () {
         before(() => {
           helper.fs.createFile('utils', 'is-string.js', fixtures.isStringV2);
         });
-        it('should update the dist', async () => {
+        it.skip('should update the dist', async () => {
```

Comment:
@urigat , this is the watch tests of the legacy. They should not be affected by your changes. If it is, please investigate why.

In general, We have two different "watchers" processes.
One for the legacy - src/consumer/component-ops/watch-components.
One for the new - extensions/workspace/watch.

Reply:
Fixed.